server/routes/user.js

```diff
@@ -4,9 +4,21 @@ import {
   updateUser,
   getUserByEmail,
   deleteUser,
+  addStripeCard,
```

Comment:
Similarly, do we need a remove card functionality?

Reply:
We will address this in a future PR where we cancel subscriptions and delete cards.